inventory/byo/hosts.origin.example

```diff
@@ -113,6 +113,9 @@ openshift_release=v3.6
 # Downgrades are not supported and will error out. Be careful when upgrading docker from < 1.10 to > 1.10.
 # docker_version="1.12.1"
 
+# Specify whether to --enable-selinux on Docker (set false if using overlay/2 storage drivers before RHEL 7.4)
```

Comment:
Hmm, instead of suggesting people to manually update their inventory (and they may be starting off an old copy of this example file), wouldn't it be possible and wise to "do the right thing" in the playbook(s) that use this value?